pkgs/development/haskell-modules/configuration-ghc-9.6.x.nix

```diff
@@ -60,6 +59,11 @@ in
   xhtml = null;
   Win32 = null;
 
+  # Was only ever released for a few exact versions of ghc
```

Comment:
But it's also only required for 9.6? Comment is a little bit confusing to me.

Reply:
The whole situation is also confusing to me really: https://github.com/NixOS/nixpkgs/pull/445672#discussion_r2430214570